capabilities/types/converters.go

```diff
@@ -125,6 +126,34 @@ var UnixTimeMilliConverter = options.NamedIntConverter("UnixTimeMilli",
 		return t.UnixMilli(), nil
 	})
 
+var BigIntConverter = options.NamedBytesConverter("BigInt",
+	func(b []byte) (big.Int, error) {
+		var negative bool
+		switch b[0] {
+		case 0:
+			negative = false
+		case 1:
+			negative = true
+		default:
+			return big.Int{}, errors.New("big int sign must be 0 or 1")
```

Comment:
would be slightly cleaner to return the `0` value here: `big.NewInt(0)`, though maybe this ends up being the same?